### ISSUE TITLE
Creating a View doesn't seem to work anymore

### ISSUE THREAD

**Issue by cskeefer** [marklogic-community/data-explorer#70]
Looks like creating a view doesn't work anymore. When trying to create one, it's returning a 404. When clicking the submit button, it is supposed to match the following rule in endpoints.xqy:

```       
<request uri="^/api/wizard/create$" endpoint="{$endpoints:API-ADHOC-WIZARD-CREATE}">
            <param name="prefix"/>
            <param name="rootElement"/>
            <param name="fileType"/>
            <param name="queryName"/>
            <param name="viewName"/>
            <param name="queryText"/>
            <param name="database"/>   
            <param name="submit"/>
            {endpoints:numbered-params("formLabelDataType", (1 to 250))}
            {endpoints:numbered-params("formLabel", (1 to 250))}
            {endpoints:numbered-params("formLabelHidden", (1 to 250))}
            {endpoints:numbered-params("formLabelIncludeMode", (1 to 250))}
            {endpoints:numbered-params("columnName", (1 to 250))}
            {endpoints:numbered-params("columnExpr", (1 to 250))}                    
            {endpoints:numbered-params("columnIncludeMode", (1 to 250))}
            <http method="POST"/>
            <http method="GET"/>
</request>
```
It doesn't match, probably due to parameters, and returns a 404. Nothing is displayed to the user, you have to check the network tab or console tab to see the 404.

**Comment by josvr** [comment]
I believe this is fixed by the CRUD Issue #13 . Please check and close this one.

**Comment by markschiffner** [comment]
Closed with FERR-90 merge
